src/pages/steps/CalendarStep/CalendarStep.tsx

```diff
@@ -245,15 +250,32 @@ const CalendarStep = ({
     messages: { calendar: t('create.toast.success.calendar') },
   });
 
+  const scrollToCalendarContainer = () => {
+    calendarStepContainer.current.scrollIntoView({
+      behavior: 'smooth',
+      block: 'center',
+    });
+  };
```

Comment:
```suggestion
  const scrollToCalendarContainer = () => {
    if (!calendarStepContainer.current) {
      return;
    }
    calendarStepContainer.current.scrollIntoView({
      behavior: 'smooth',
      block: 'center',
    });
  };
```

Since the ref can be null in the beginning